File: RestaurantApp/RestaurantAppServer/Domain/Entities/OrderItem.cs
using System.ComponentModel.DataAnnotations;

namespace Domain.Entities;

public class OrderItem
{
    [Key]
    public int Id { get; set; }
    public int Quantity { get; set; }

    public int UserId { get; set; }
    
    public int DishId { get; set; }
    public Dish Dish { get; set; }
}

File: RestaurantApp/RestaurantApp/RestaurantApp/Models/OrderItem.swift
//
//  OrderItem.swift
//  RestaurantApp
//
//  Created by Азат Зиганшин on 05.05.2024.
//

import Foundation

struct OrderItem: Codable {
    let id: Int
    let quantity: Int
    let userId: Int
    let dishId: Int
    let dish: Dish
}
